cli/valet.php

```diff
@@ -226,8 +226,8 @@
     /**
      * Echo the currently tunneled URL.
      */
-    $app->command('fetch-share-url', function () {
-        output(Ngrok::currentTunnelUrl());
+    $app->command('fetch-share-url [domain]', function ($domain = null) {
+        output(Ngrok::currentTunnelUrl($domain ?: Site::host(getcwd()).'.'.Configuration::read()['tld']));
```

Comment:
Would this struggle if someone's parked it as something else?

Reply:
If they've linked it to something other than the same as the foldername, then the `Site::host()` function resolves the link and uses that instead.

https://github.com/laravel/valet/blob/52b3b47db8ea0f77dee00af3a6c78a7c810cc597/cli/Valet/Site.php#L62-L77

Reply:
brilliant